src/utils/DataFetching/Sorting/usePercentilesOrZScores.ts

```diff
@@ -0,0 +1,6 @@
+export const usePercentilesOrZScores = (category: any, usePercentiles: boolean): any[] | undefined => {
+    if (category) {
+        return usePercentiles ? category.percentileDatasetValues : category.zScoreDatasetValues;
+    }
+    return undefined;
+};
```

Comment:
Could concider removing this file as it dosnt do that much. Just move the logic to GrowthChart.tsx